src/config/logger.ts

```diff
@@ -17,34 +17,47 @@ const outputMode = process.env.MICROS_ENV ? "json" : "short";
 //
 export const FILTERING_FRONTEND_HTTP_LOGS_MIDDLEWARE_NAME = "frontend-log-middleware";
 // add levelInString to include DEBUG | ERROR | INFO | WARN
-const LOG_STREAM = filteringHttpLogsStream(FILTERING_FRONTEND_HTTP_LOGS_MIDDLEWARE_NAME,
+const LOG_STREAM = filteringHttpLogsStream(
+	FILTERING_FRONTEND_HTTP_LOGS_MIDDLEWARE_NAME,
 	bformat({ outputMode, levelInString: true })
 );
 
-const responseSerializer = (res: AxiosResponse) => ({
+const responseSerializer = (res) => res && ({
 	...stdSerializers.res(res),
-	config: res?.config,
+	config: res.config,
 	request: requestSerializer(res.request)
 });
 
-const requestSerializer = (req: Request) => (!req || !req.socket) ? req : {
+const requestSerializer = (req) => req && ({
 	method: req.method,
 	url: req.originalUrl || req.url,
 	path: req.path,
 	headers: req.headers,
 	remoteAddress: req.socket.remoteAddress,
 	remotePort: req.socket.remotePort,
```

Comment:
won't this run into an `undefined` error when `req.socket` is not there? That was specifically handled in the previous version of the code

Reply:
yep, good catch.